packages/eslint-config/index.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: './tsconfig.json',
+    project: ['./tsconfig.json', './packages/**/tsconfig.json'],
```

Comment:
My editor had been complaining about this. I'm very uncertain why ci didn't complain, but my editor did.

Reply:
It caused the first few errors as I didn't see them like I expected to.